referrer-policy/no-referrer-policy/no-referrer-policy.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE HTML>
+<html>
+  <head>
+    <title>Check that subresource gets the referrer URL when no explicit
```

Comment:
Just to follow up on the current implementation. I'm actually using multiple frames - one per scenario. The order in which they load does not matter since each of them receive an ID and the message listener is mapping the appropriate test to run via that ID.  Should running independent tests in a particular order ever matter? As for the hanging, is it more likely to have an iframe hang if it's on the same page of the same test or if it's loaded once per test html file (popup)?

I suppose it would be a good idea to split the test to have only one per file (html page). However, that could be done easily if I continue to introduce more structure into the tests and as that structure emerges. At this point it's easier to keep them in one place while I'm still thinking up new scenarios for the referrer-policy and as I read more of the specs.

I plan to do plenty of refactoring for reusability as things evolve.